src/amulware.Graphics/Core/Color.cs

```diff
@@ -524,25 +521,28 @@ public float Hue
         {
             get
             {
-                float r = this.r / 255f;
-                float g = this.g / 255f;
-                float b = this.b / 255f;
+                var floatR = r / 255f;
+                var floatG = g / 255f;
+                var floatB = b / 255f;
 
                 float h;
 
-                var max = Math.Max(r, Math.Max(g, b));
+                var max = Math.Max(floatR, Math.Max(floatG, floatB));
+                // ReSharper disable once CompareOfFloatsByEqualityOperator
```

Comment:
Disable for the entire block and enable again at the end?

Reply:
I can see why that is nicer. I'll change it, but not without pointing out my reasoning for doing it this way: I still think the warning is valuable, and if there is ever new float logic added within the block that _shouldn't_ compare floats by equality, we'll lose the warning by disabling it for the whole block. It's a trade-off, but in this case I think it's fine.

Reply:
I fully agree with your thinking, but I think in this case it's nicer. It's very unlikely this code will be changed significantly.